api/res/openapi.yml

```diff
@@ -246,7 +246,7 @@ paths:
           example: '2019-07-22T08:42:47-00:00'
         - in: query
           name: limit
-          description: The number of log lines to retrieve. If not present, 1000 lines will be retrieved.
+          description: The number of log lines to retrieve. If not present, all the lines from `since` are retrieved.
```

Comment:
The new query parameter and the new content-type is not yet included here.

Reply:
Offering the event-stream as it is in openapi.yml is not straightforward. There are a few ways to do it, I will confirm the applicable one and do it in the next hour.